### ISSUE TITLE
ConcatDataset does not seem to support customize dataset settings "metainfo" yet

### ISSUE THREAD

**Issue by Alin1102** [open-mmlab/mmdetection#10499]
Here is the error log when I try to run semi detection config : soft-teacher on my customize dataset:

**Comment by Czm369** [comment]
Yep, this is the issue of `ConcatDataset`, you should keep `metainfo` consistent between two datasets.
